inc/classes/class-genesis-featured-widget-amplified.php

```diff
@@ -1220,27 +1198,27 @@ function gfwa_do_post_meta( $instance ) {
 }
 
 /**
- * Returns "display: none;" if option and value match, or of they don't match with $standard is set to false
+ * Returns "display: none;" if option and value match, or if they don't match with $standard set to false.
  *
  * @author Nick Croft
  * @since 0.8
  * @version 0.8
  * @param array   $instance Values set in widget $instance.
  * @param mixed   $option instance option to test.
  * @param mixed   $value value to test against.
- * @param boolean $standard echo standard return false for oposite.
+ * @param boolean $standard echo standard return false for opposite.
  * @return string
  */
 function gfwa_get_display_option( $instance, $option = '', $value = '', $standard = true ) {
 	$display = '';
 	if ( is_array( $option ) ) {
 		foreach ( $option as $key ) {
-			if ( in_array( $instance[ $key ], $value ) ) { // WPCS: loose comparison ok.
+			if ( in_array( $instance[ $key ], $value, false ) ) {
```

Comment:
There are instances where this doesn’t work with strict. Loose comparison needs to be allowed or the settings can break.

Reply:
Absolutely, and this is maintained. The third arg is `false` (to the question "should this be a strict comparison?") and not `true`.

By default, WPCS complains that this should be strict, but when that can't be done, we can pass an explicit `false`, and then [WPCS uses a different error code](https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/blob/cc7d15d9de07502ead2c4515dd0c3007d62dacaf/WordPress/Sniffs/PHP/StrictInArraySniff.php#L86-L92) - and that is subsequently excluded within the `.phpcs.xml.dist` in this PR so that it is not reported.

Unlike the strict comparison (`==` vs `===`) a few rows down, which does [support](https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/blob/cc7d15d9de07502ead2c4515dd0c3007d62dacaf/WordPress/Sniffs/PHP/StrictComparisonsSniff.php#L51) a whitelist flag (`// WPCS loose comparison ok.`), there is no such support for a whitelist flag in the StrictInArray sniff.

Reply:
Sorry I misremembered the true/false argument. 

Interestingly the flag I added cleared the WPCS warning for me with the version I am running locally.

Reply:
> with the version I am running locally

And that's the problem with global dependencies - I've got no idea what version of PHPCS or WPCS you're using.

With development dependencies [explicitly declared and isolated](https://12factor.net/dependencies) as part of the package, we're all working from the same page.